Gemfile

```diff
@@ -25,6 +25,9 @@ gem 'cancancan'
 gem 'rgeo'
 gem 'rgeo-geojson'
 gem 'rails_12factor', group: :production
+gem 'high_voltage', github: 'thoughtbot/high_voltage'
+gem 'georuby', require: ['georuby', 'geo_ruby/ewk']
```

Comment:
oh wierd, I looked through the gem file to see if one or the other was in there and didn't some how it went right by me